build.sbt

```diff
@@ -1,40 +1,66 @@
 
-name := "scala-enum"
-
-organization := "me.jeffmay"
-
-version := "1.0.0"
-
-crossScalaVersions := Seq("2.11.6", "2.10.4")
-
-scalacOptions := {
-  // the deprecation:false flag is only supported by scala >= 2.11.3, but needed for scala >= 2.11.0 to avoid warnings
-  CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, scalaMinor)) if scalaMinor >= 11 =>
-      // For scala versions >= 2.11.3
-      Seq("-Xfatal-warnings", "-deprecation:false")
-    case Some((2, scalaMinor)) if scalaMinor < 11 =>
-      // For scala versions 2.10.x
-      Seq("-Xfatal-warnings")
-  }
-} ++ Seq(
-  "-feature",
-  "-Xlint",
-  "-Ywarn-dead-code",
-  "-encoding", "UTF-8"
+lazy val commonRootSettings = Seq(
+  version := "1.1.0",
+  scalaVersion := "2.11.6",
+  crossScalaVersions := Seq("2.11.6", "2.10.4"),
+  organization := "me.jeffmay",
+  organizationName := "Jeff May"
+) ++ bintraySettings ++ bintrayPublishSettings
+
+commonRootSettings
```

Comment:
What's the effect of this line?  Isn't `commonRootSettings` referenced in the very next expression?